### PR TITLE
Errors and validation

### PR DESCRIPTION
**Pull request by LHinson**
Adding documentation for errors and validation as seen in google docs here: https://docs.google.com/document/d/1UPx4smbKUsbyzL84LoEGlCyI5u89nPNs4cd2I0x4-WM/edit#

### CODE REVIEW

**Comment by LHinson** [comment]
@lclay2 FYI

File: pattern-library/forms-and-controls/form-field-layouts/design/overview.md [hunk 1]
```diff
@@ -20,8 +20,8 @@ A consistent left edge, along with close proximity of label and input, offers ea
 ![Top Aligned Labels Viewport Recommendations](top-aligned-viewport.png)
```
Comment (andresgalante) [inline pattern-library/forms-and-controls/form-field-layouts/design/overview.md]: This image is missing

File: pattern-library/forms-and-controls/input-controls/design/overview.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Text input fields are one type of input control. They are extremely common form elements, providing a useful foundation for form field recommendations. 
+
+![Right Aligned Labels](img/right-aligned-labels.png)
+
+## Text Field Inputs
+* Always provide a label.  Do not include a colon at the end of the label. Visit the Capitalization section of Terminology and Wording for information regarding capitalization.
+* Pre-populate fields with known values when possible (eg. default IP address.)
+* When presented in a form, the initial input field is usually the first input field which is required. The primary input field should receive focus, when the form is presented to the user. 
+
+### Notes about keyboard interactions:
```
Comment (andresgalante) [inline pattern-library/forms-and-controls/input-controls/design/overview.md]: Do we use colons after the title?

**Comment by andresgalante** [comment]
@LHinson Other than the missing image and the wrong path, the text looks ok.
Great job! Let me know if I can help with the images.

**Comment by andresgalante** [comment]
@LHinson since we've merge @bleathem PR, this PR has conflicts.
Let me know if I can help resolve them

**Pull request merged**